build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+import org.zaproxy.gradle.addon.AddOnStatus
+import org.zaproxy.gradle.addon.manifest.tasks.ConvertChangelogToChanges
+
+plugins {
+    `java-library`
+    eclipse
+    id("org.zaproxy.add-on") version "0.1.0"
+}
+
+eclipse {
+    classpath {
+        // Prevent compilation of zapHomeFiles.
+        sourceSets = listOf()
+    }
+}
+
+version = "1"
+description = "FuzzDB web backdoors which can be used with the ZAP fuzzer"
+
```

Comment:
Lets insert here:
```kts
java {
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
}
```
(The add-on doesn't have Java source, but better specify it anyway.)